media/js/firefox/feedback-ga-tracking.js

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+$(function () {
+    'use strict';
+
+    function trackGenericLink(action, href, callback) {
+        if (typeof callback === 'function') {
+            gaTrack(['_trackEvent', 'Feedback Interactions', action, href], callback);
+        } else {
+            gaTrack(['_trackEvent', 'Feedback Interactions', action, href]);
+        }
+    }
+
+    // track clicks in the secondary CTAs
+    $('.action-secondary a').on('click', function(e) {
```

Comment:
Small nit-pick on these event listeners - could be slightly more efficient to delegate, like:

`$('.action-secondary').on('click', 'a', function(e) {`

This creates a single event listener on the container, instead of an event listener on each `<a>` tag.